charts/snyk-broker/templates/secrets.yaml

```
@@ -38,6 +38,16 @@ data:
   "{{ .Values.scmType}}-token-key": {{ .Values.bitbucketPassword | b64enc | quote }}
 ---
 {{- end }}
+{{- if .Values.bitbucketPat }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.scmType}}-token{{if not .Values.disableSuffixes }}-{{ .Release.Name }}{{ end }}
+type: Opaque
+data:
+  "{{ .Values.scmType}}-token-key": {{ .Values.bitbucketPat | b64enc | quote }}
```

@aarlaud, we could use `stringData` instead of `data`, so b64enc will not be needed, wdyt?
```suggestion
stringData:
  "{{ .Values.scmType}}-token-key": {{ .Values.bitbucketPat | quote }}
```